src/track/keyutils.cpp

```diff
@@ -763,3 +767,34 @@ int KeyUtils::keyToCircleOfFifthsOrder(mixxx::track::io::key::ChromaticKey key,
         return s_sortKeysCircleOfFifthsLancelot[static_cast<int>(key)];
     }
 }
+
+// static
+QVariant KeyUtils::keyFromColumns(QVariant&& rawValue, QVariant&& keyCodeValue) {
+    // Helper function used by basetrackcache.cpp and basetracktablemodel.cpp
+    // to determine the Key string from either the LIBRARYTABLE_KEY or the
+    // LIBRARYTABLE_KEY_ID field.
+    //
+    // If we know the semantic key via the LIBRARYTABLE_KEY_ID
+    // column (as opposed to the string representation of the key
+    // currently stored in the DB) then lookup the key and render it
+    // using the user's selected notation.
+    if (keyCodeValue.isNull()) {
+        // Otherwise, just use the column value as is
+        return std::move(rawValue);
```

Comment:
```suggestion
QVariant KeyUtils::keyFromColumns(const QVariant& rawValue, const QVariant& keyCodeValue) {
    // Helper function used by basetrackcache.cpp and basetracktablemodel.cpp
    // to determine the Key string from either the LIBRARYTABLE_KEY or the
    // LIBRARYTABLE_KEY_ID field.
    //
    // If we know the semantic key via the LIBRARYTABLE_KEY_ID
    // column (as opposed to the string representation of the key
    // currently stored in the DB) then lookup the key and render it
    // using the user's selected notation.
    if (keyCodeValue.isNull()) {
        // Otherwise, just use the column value as is
        return rawValue;
```